common/osu/test_difficultycalculator.py

```diff
@@ -235,13 +235,13 @@ class TestDifficalcyPerformancePlusDifficultyCalculator:
     def test_enigne(self):
         assert (
             DifficalcyPerformancePlusDifficultyCalculator.engine()
-            == "https://github.com/Syriiin/osu"
+            == "https://github.com/Syriiin/osu/tree/performanceplus"
```

Comment:
we need to run an query to update the pp+ engine package name to avoid a full db recalc and orphaned rows